students/PeterB/Session2/series.py

```diff
@@ -0,0 +1,101 @@
+# fibonacci number = 0,1,1,3,5,8,13, 21
+
+def fibonacci(n):
+    a,b = 0, 1
+    for i in range(n-1):
+        a, b = b, a+b
+    return a
```

Comment:
almost right -- it's supposed to start with 0, 1, 1, 2, ....

butyou get an extra 0:

```
In [2]: def fibonacci(n): 
   ...:     a,b = 0, 1 
   ...:     for i in range(n-1): 
   ...:         a, b = b, a+b 
   ...:     return a 
   ...:                                                                         

In [3]: fibonacci(0)                                                            
Out[3]: 0

In [4]: fibonacci(1)                                                            
Out[4]: 0

In [5]: fibonacci(2)                                                            
Out[5]: 1
```

Reply:
and by returning a, you are computing a final b that you don't use. not a big deal here, but in generaly, you don't want to compute anything extra.